package.json

```diff
@@ -101,7 +101,6 @@
     "react-copy-to-clipboard": "4.3.1",
     "react-dom": "15.6.2",
     "react-dropzone": "3.13.4",
-    "react-foundation-components": "git+https://github.com/valzav/react-foundation-components.git#d14362c7c8eee946a4acc3b18d70271d5a82813e",
```

Comment:
we also need to remove `foundation-sites` as well as the reference to it in `src/app/assets/stylesheets/app.scss`

Reply:
I think we still use foundation-sites css. This is just removing the 'react-foundation-components' library.

I think I might be remiss in suggesting we should remove foundation-sites.

Reply:
A seperate issue might be to just mode the css we're using in there over into condenseer, but I think that's a seperate issue...